doc/distrib/NodeHelpFiles/DSRevitNodesUI.ElementById.md

```diff
@@ -0,0 +1,6 @@
+## In Depth
+Given an element id (int), or a unique id (string), Element By Id will select the related element from the current document (file).
```

Comment:
Ideally present tense instead of future, so "selects the related element" instead of "will select the related element"